### PR TITLE
Bug 1192752 - Identity popup checks in security tests are failing due to behavior changes

### PR DESCRIPTION
**Pull request by whimboo**

### CODE REVIEW

**Comment by whimboo** [comment]
The failures as listed by travis are located in the privatebrowsing test. This is different and will be fixed next.

File: firefox_puppeteer/ui/toolbars.py [hunk 1]
```diff
@@ -558,20 +558,30 @@ def secure_connection_label(self):
...
+                                         'button[when-mixedcontent=active-blocked]')
 
     @property
-    def internal_connection_label(self):
-        """The DOM element which represents the identity popup internal connection label.
+    def enable_mixed_content_blocking_button(self):
+        """The DOM element which represents the enable mixed content blocking button.
 
-        :returns: Reference to the identity-popup internal connection label.
+        :returns: Reference to the ebnable mixed content blocking button.
```
Comment (jmaher) [inline firefox_puppeteer/ui/toolbars.py]: typo for enable here

File: firefox_ui_tests/functional/security/test_no_certificate.py [hunk 2]
```diff
@@ -49,8 +49,7 @@ def test_no_certificate(self):
         Wait(self.marionette).until(lambda _: self.identity_popup.is_open)
 
         # Check the idenity popup doorhanger
-        self.assertEqual(self.identity_popup.element.get_attribute('className'),
-                         'unknownIdentity', 'The Larry UI is unknown (aka Grey)')
+        self.assertEqual(self.identity_popup.element.get_attribute('connection'), 'not-secure')
```
Comment (jmaher) [inline firefox_ui_tests/functional/security/test_no_certificate.py]: do we loose coverage for the larry ui?

Comment (whimboo) [inline firefox_ui_tests/functional/security/test_no_certificate.py]: It's totally the same check here. The `className` simply doesn't exist anymore, and is covered via `connection` now.

**Pull request closed without merging**